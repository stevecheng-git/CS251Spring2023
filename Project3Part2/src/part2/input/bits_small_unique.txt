12
29
6
24
22
5
25
0
10
20
23
13
28
21
8
26
4
15
7
18
9
31
2
27
3
30
14
11
16
1
17
19
