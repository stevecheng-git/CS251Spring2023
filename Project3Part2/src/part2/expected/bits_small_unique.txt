0 16 0 0
0 16 0 32
64 16 0 32
64 16 0 33
64 16 64 33
96 16 64 33
96 16 64 35
97 16 64 35
97 20 64 35
97 20 80 35
97 20 -48 35
97 52 -48 35
97 52 -48 51
97 52 -16 51
97 53 -16 51
97 53 -16 55
113 53 -16 55
113 -75 -16 55
-15 -75 -16 55
-15 -75 -12 55
-15 -73 -12 55
-15 -73 -12 -73
-11 -73 -12 -73
-11 -73 -12 -65
-3 -73 -12 -65
-3 -73 -12 -1
-3 -9 -12 -1
-3 -1 -12 -1
-3 -1 -11 -1
-1 -1 -11 -1
-1 -1 -9 -1
-1 -1 -1 -1
